packages/@textlint/kernel/package.json

```diff
@@ -15,11 +15,7 @@
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
-    "test": "npm-run-all test:prepare test:unit",
-    "test:prepare": "npm-run-all test:build test:cp_mocha",
-    "test:build": "cross-env NODE_ENV=test tsc -p ./test",
-    "test:cp_mocha": "cpx 'test/**/mocha.opts' 'out/test/' --preserve --verbose",
-    "test:unit": "mocha out/test",
+    "test": "mocha test",
     "clean": "rimraf out/ lib/ es_modules/",
```

Comment:
`es_modules` can be removed when this PR is completed.

Reply:
Additionaly, need to change `tsconfig.json` in https://github.com/textlint/textlint/blob/4143fd3794bcbdef4bcc5d13f2c50f90b7870502/packages/%40textlint/kernel/tsconfig.json#L7

```diff
- "outDir": "./es_modules/",
+ "outDir": "./lib/",
```

Previously,

1. `tsc`: `src` -> `es_modules`
2. `babel`: `es_modules` -> `lib`

This PR(Future)

1. `tsc`: `src` -> `lib`

Reply:
Should I remove `es_modules` in this PR or other PR ?

Reply:
This PR.

Reply:
Certainly.